draft-ietf-dnsop-svcb-https.md

```diff
@@ -193,7 +197,7 @@ hostname as the `host`.
 * The "service name" is the `host` portion of the authority.
 * The "authority endpoint" is the authority's hostname and a port number implied
   by the scheme or specified in the URI.
-* An "alternative endpoint" is a hostname, port number, and other associated
+* An "alternative endpoint" is a domain name, port number, and other associated
```

Comment:
Erm. This is a larger change than we agreed to in https://mailarchive.ietf.org/arch/msg/dnsop/LhEufm0a8IjoXZ_ck0b9RTSkDTc/

Reply:
This is fixing this to be consistent with elsewhere in the document (so an error/errata here), so is arguably not a technical change?  (Should we ask the list about this in-particular?)
Elsewhere we have in the draft we are clear that TargetName is a domain name:

> 2. TargetName: The *domain name* of either the alias target (for
>   AliasMode) or the alternative endpoint (for ServiceMode).

> TargetName is a `<domain-name>` ({{!RFC1035, Section 5.1}}),

> In AliasMode, the TargetName will be the name of a domain that resolves to SVCB,